cmd/manager/main.go

```diff
@@ -47,15 +50,26 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	utilruntime.Must(authv1alpha1.AddToScheme(scheme))
+
 	//+kubebuilder:scaffold:scheme
 }
 
 func main() {
+	featureGates := featuregate.NewFeatureGate()
+	features := make(map[featuregate.Feature]featuregate.FeatureSpec)
+	features[featuregate.Feature("EphemeralIdentity")] = featuregate.FeatureSpec{
```

Comment:
how about we move the feature definition to a dedicated package? e.g.:

https://github.com/kubernetes-sigs/kubefed/blob/master/pkg/features/features.go